### PR TITLE
Feature/index page announcements

### PR DESCRIPTION
**Pull request by nicole-cranon**
Could you guys review the changes of this branch and see if it is ready to merge?

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,20 @@
+0 info it worked if it ends with ok
```
Comment (Wmaxlees) [inline npm-debug.log]: Can you remove this file? We don't want extra log files in the repo if we don't have to.

**Comment by Wmaxlees** [comment]
Can we move the folder from `site/public/acmApp/components/announcements/` to `site/public/acmApp/components/articles/` for consistency. Sorry, that's Rob and my fault. We've been jumping back and forth between article and announcement in the project.

**Comment by nicole-cranon** [comment]
I will remove the log file and move to articles directory.

**Comment by robfitzgerald** [comment]
had to modify the following files for it to run when i built it from scratch:

```
    modified:   site/public/acmApp/acmApp.routes.js
    modified:   site/public/acmApp/partials/index.html
    modified:   site/views/layout.jade
```

i had to replace announcements with articles
and i changed gitignore so it ignores *.log (for good measure).
may i push these changes to your branch, nicole?

**Comment by nicole-cranon** [comment]
Please do

**Comment by robfitzgerald** [comment]
so @nicranonn as long as you're ok with my changes then this pull request gets my :+1: and we can let @Wmaxlees close it out.

**Comment by nicole-cranon** [comment]
Sweet! I feel all accomplished for my little contribution

**Comment by robfitzgerald** [comment]
:fireworks:  yayyyy :fireworks: 
...
:fire_engine: ?

**Comment by nicole-cranon** [comment]
😂

**Pull request merged**